plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1319,13 +1319,13 @@ protected void setupMemoryBalloonStatsPeriod(Connect conn) {
         }
     }
 
-    private void enableSSLForKvmAgent(final Map<String, Object> params) {
+    private void enableSSLForKvmAgent() {
```

Comment:
removing the parameter means that the passphrase can no longer come from the MS. It must be in the properties file now. Is that the intention, or should we try both?

Reply:
@DaanHoogland 
keystore.passphrase is not in `params` now, but it is saved in `agent.properties`
we just need to load the property from `agent.properties`, same as other properties

Reply:
btw, the keystore passphrase of this method was never passed from management server.
it is loaded from agent.properties in 4.17.x.x (it was not broken)

Reply:
@weizhouapache, you mean it never worked then?

Reply:
@GutoVeronezi 
it worked well in 4.17.0.0, 4.17.1.0, 4.17.2.0

Reply:
@GutoVeronezi 
it might be caused by this change in #6348

![image](https://github.com/apache/cloudstack/assets/57355700/1746e965-efdb-4fd1-88aa-507c8ad7421f)

Reply:
I see. Before instantiating `Agent`, `AgentShell#Init` loads the properties from `agent.properties`; as we are not using `_shell.getProperties()` anymore, the map passed to `LibvirtComputingResource#enableSSLForKvmAgent` does not have the passphrase anymore. Makes sense.